src/wallet/rpcwallet.cpp

```diff
@@ -3389,9 +3655,9 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
     if (!EnsureWalletIsAvailable(fHelp))
         return NullUniValue;
 
-    if (fHelp || params.size() < 2 || params.size() > 4)
+    if (fHelp || params.size() < 2 || params.size() > 5)
         throw runtime_error(
-            "z_sendmany \"fromaddress\" [{\"address\":... ,\"amount\":...},...] ( minconf ) ( fee )\n"
+            "z_sendmany \"fromaddress\" [{\"address\":... ,\"amount\":...},...] ( minconf ) ( fee ) ( revealamount )\n"
```

Comment:
```suggestion
            "z_sendmany \"fromaddress\" [{\"address\":... ,\"amount\":...},...] ( minconf ) ( fee ) ( revealAmount )\n"
```

Reply:
I think that all-lowercase is consistent with CLI arguments in other cases?

Reply:
We don't use a consistent convention for this unfortunately; it's sometimes alloneword and sometimes lowerCamelCase.